packages/react/src/utils/versions.ts

```diff
@@ -17,6 +17,7 @@ export const typesReactIsVersion = '19.0.0';
 export const reactViteVersion = '^4.2.0';
 
 export const typesNodeVersion = '18.16.9';
+export const typesReactRouterNodeVersion = '^20.0.0';
```

Comment:
so now we will generate apps with 2 node version, without react router, it will be at version 18, with reach router, it will be at version 20. i am just wondering, whether it will easier just to upgrade node to 20 for all.

Reply:
It is possible that we could just upgrade to node 20 the LTS is currently node v22. 
This is types as well so might be a good option to update it.